pdns/recursordist/rec-main.cc

```diff
@@ -104,7 +104,8 @@ std::shared_ptr<Logr::Logger> g_slogudpin;
 std::shared_ptr<Logr::Logger> g_slogudpout;
 
 /* without reuseport, all listeners share the same sockets */
-deferredAdd_t g_deferredAdds;
+static deferredAdd_t g_deferredAdds;
+static deferredAdd_t g_deferredTCPAdds;
```

Comment:
Just a nit but the `g_` prefix usually indicates globals, so we might want to rename these.
```suggestion
static deferredAdd_t s_deferredAdds;
static deferredAdd_t s_deferredTCPAdds;
```